js/private/js_binary.bzl

```diff
@@ -168,6 +168,19 @@ _ATTRS = {
         values = _LOG_LEVELS.keys(),
         default = "error",
     ),
+    "patch_node_fs": attr.bool(
```

Comment:
Should we try to put the word "sandbox" in the name since "patch" normally implies a bug-fix?

Reply:
sandbox wouldn't be accurate either since the guards prevent leaving execroot, runfiles and the sandbox. I create a pretty verbose docstring on this one. most users will never need to touch it but it does leave an easy opt-out if needed.

Reply:
`node_fs_guards` perhaps but I like "patch" as it is technically what we are doing by changing the behavior of the node fs code

Reply:
I guess I'd worry that `fs` will become overloaded and will mean patching a lot more then the node "fs" library like the name implies 🤷

Reply:
Hopefully we won't be patching much more in node so we won't to worry about it :)

Reply:
🐒 -patch